.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 
 node_js:
-  - iojs
   - 0.12
-  - 0.10
+  - iojs
```

Comment:
perhaps we want to follow [node's LTS](https://github.com/nodejs/LTS) for our matrix. today that would be 0.10, 0.12, and 4